puppet/modules/freight/manifests/user.pp

```diff
@@ -8,6 +8,8 @@
 ) {
   require freight
 
+  ensure_packages(['ruby'])
```

Comment:
Does it make sense to add `require => Package['ruby']` to `file { "/etc/cron.daily/${user}":` so we know why it is we need it? It also means we don't deploy it unless Ruby could actually be installed.

Reply:
Probably

Reply:
done